templates/init/functions/javascript/package.lint.json

```diff
@@ -3,6 +3,7 @@
   "description": "Cloud Functions for Firebase",
   "scripts": {
     "lint": "./node_modules/.bin/eslint .",
+    "build": "",
```

Comment:
I'm not sure what build script needs to be here, but the predeploy breaks with an empty string so I've left this here for now